tests/integration/helpers.bash

```diff
@@ -476,6 +477,18 @@ function teardown_recvtty() {
 	rm -f "$CONSOLE_SOCKET"
 }
 
+function setup_seccompagent() {
+	"$SECCOMPAGENT" >/dev/null 2>&1 &
```

Comment:
Maybe call it with the `-socketfile="$BATS_TMPDIR/seccompagent.socket"` to avoid bad interaction with already running seccomp agents.